.luacheckrc

```diff
@@ -19,6 +19,8 @@ cache = true
 ignore = {
   "631",  -- max_line_length
   "212/_.*",  -- unused argument, for vars with "_" prefix
+  "121", -- setting read-only global variable 'vim'
+  "122", -- setting read-only field of global variable 'vim'
```

Comment:
Will this allow cases that should be caught? Can we special case `vim` with something similar to `read_globals` (just below here)?

Reply:
We probably shouldn't set it as a read_global, but as a global. Then we dont need to set the ignore for 121 and 122.

Global is better, since that is exactly what we want.